app/services/head-data.js

```diff
@@ -25,6 +25,11 @@ export default HeadData.extend({
   }),
 
   slug: computed('routeName', function() {
+    // if there is no current model
+    if (!this.currentRouteModel) {
+      return null;
+    }
+    
     if(this.currentRouteModel.id === 'index') {
```

Comment:
May be have this as another `or ` computed prop so that you skip the above null check?